pkg/platform/platform.go

```diff
@@ -82,6 +82,9 @@ func (platform *Platform) GetCA() certs.CertificateAuthority {
 func readCA(ca *types.CA) (*certs.Certificate, error) {
 	cert := files.SafeRead(ca.Cert)
 	privateKey := files.SafeRead(ca.PrivateKey)
+	if ca.Password == "" {
```

Comment:
This should probably go into certs.DecryptCertificate that attempts to just decode the cert if the password is empty

Reply:
This should probably log a warning as well.

Reply:
Then I should change it inside `flanksource/commons`